mesa/experimental/jupyter_viz.py

```diff
@@ -51,6 +51,20 @@ def make_model():
         set_current_step(0)
         return model
 
+    def calculate_space_size():
+        pass
+
+    def calculate_agent_size():
```

Comment:
The Matplotlib marker size shouldn't depend on the number of agents. It should depend only on the grid size and the computer's screen size.

Reply:
It should be more abstract, like screen width, because in a Jupyter notebook, the width should be even smaller.

Reply:
hmm, let me think how this can be implemented. Currently the size of each agent is parsed via `agent_portrayal` and hence it's easier(also less compute expensive) to manipulate.

Reply:
I think it would also be important to show all agents in a smaller window size, hence the implementation can work there. I try to come up with some scaling to fit the screen width.